e2e/sendSheetFlow.spec.js

```diff
@@ -57,20 +57,12 @@ describe('Send Sheet Interaction Flow', () => {
   // });
 
   it('Should show all wallet sections', async () => {
-    await Helpers.delay(1000);
+    await Helpers.delay(5000);
     await Helpers.checkIfElementByTextIsVisible('Pools');
     await Helpers.swipe('wallet-screen', 'up');
     await Helpers.checkIfElementByTextIsVisible('Collectibles');
   });
 
-  it('Should say "poopcoin.eth" in the Profile Screen header', async () => {
-    await Helpers.delay(1000);
-    await Helpers.swipe('wallet-screen', 'right');
-    await Helpers.delay(2000);
-    await Helpers.checkIfElementByTextIsVisible('poopcoin.eth');
-    await Helpers.swipe('profile-screen', 'left');
```

Comment:
Can you keep this one and make sure it shows the address of the test wallet instead?
The reason I did it was to catch the "initializing..." bug when the wallet is created incorrectly